src/metakb/main.py

```diff
@@ -78,7 +78,7 @@ def custom_openapi() -> dict:
     response_description=search_study_response_descr,
     description=search_studies_descr,
 )
-async def get_studies(
+async def get_studies_intersect(
```

Comment:
now feeling like maybe elongating the method name is unnecessary with actual search behavior ironed out and more explicit docstrings. idk. just seems like a long name.